### ISSUE TITLE
[@uifabric/example-app-base] Aria-expanded attribute is not defined for "Settings" button 

### ISSUE THREAD

**Issue by tyveille** [microsoft/fluentui#16550]
### Environment Information

Test Environments:
Version:  2004 (OS Build: 20236.1005)
Browser: Edge chromium 
Tool: Developer tool

### Describe the issue:

User Experience:
When navigating to a site using example-app-base, the screen reader will not able to know the state of the control and might miss the functionality present under the control if aria-expanded attribute is not defined for the control.

### Please provide a reproduction of the issue in a codepen:

Unable to use codepen as the issue is located in @uifabric/example-app-base

Navigate on "Settings" button
Inspect it through developer tool
Verify the issue

#### Actual behavior:

Aria-expanded attribute is not defined for "Settings" button due to which screen reader is announcing "Setting button" only.

#### Expected behavior:

Aria-expanded attribute should be defined for "Settings" button.

### Documentation describing expected behavior

N/A

**Comment by JustSlone** [comment]
@tyveille can you provide a bit more details as to where this issue presents itself? Having trouble figuring out where the "Settings" button you are referring to is.
